docker-compose.yml

```diff
@@ -120,9 +117,113 @@ x-hierarchy:
   # helper services
   - impala
   - postgres
+  - python-wheel-manylinux-2010
+  - python-wheel-manylinux-2014
+  - python-wheel-manylinux-test
+  - python-wheel-windows-vs2017
+  - python-wheel-windows-test
+
+volumes:
+  conda-ccache:
+    name: ${ARCH}-conda-ccache
+  debian-ccache:
+    name: ${ARCH}-debian-${DEBIAN}-ccache
+  ubuntu-ccache:
+    name: ${ARCH}-ubuntu-${UBUNTU}-ccache
+  fedora-ccache:
+    name: ${ARCH}-fedora-${FEDORA}-ccache
+  debian-rust:
+    name: ${ARCH}-debian-${DEBIAN}-rust
+  maven-cache:
+    name: maven-cache
+  wheel-manylinux1-ccache:
+    name: centos-python-manylinux1-ccache
+  wheel-manylinux2010-ccache:
+    name: centos-python-manylinux2010-ccache
+  wheel-manylinux2014-ccache:
+    name: centos-python-manylinux2014-ccache
+  wheel-windows-clcache:
+    name: windows-wheel-clcache
 
 services:
 
+  # See available versions at:
+  #    https://quay.io/repository/pypa/manylinux2010_x86_64?tab=tags
+  python-wheel-manylinux-2010:
+    image: ${REPO}:python-${PYTHON}-wheel-manylinux-2010
+    build:
+      args:
+        base: quay.io/pypa/manylinux2010_x86_64:2020-12-03-912b0de
+        vcpkg: ${VCPKG}
+        python: ${PYTHON}
+      context: .
+      dockerfile: ci/docker/wheel-manylinux-201x.dockerfile
+      cache_from:
+        - ${REPO}:python-${PYTHON}-wheel-manylinux-2010
+    environment:
+      <<: *ccache
+      MANYLINUX_VERSION: 2010
+    volumes:
+      - .:/arrow:delegated
+      - ${DOCKER_VOLUME_PREFIX}wheel-manylinux2010-ccache:/ccache:delegated
+    command: /arrow/ci/scripts/wheel_manylinux_build.sh
```

Comment:
How about prepend `python_` because `python-` exists in related names?

Reply:
Updated.